be/src/storage/tablet_reader.h

```diff
@@ -49,11 +54,12 @@ class TabletReader final : public ChunkIterator {
     using PredicateList = std::vector<const ColumnPredicate*>;
     using PredicateMap = std::unordered_map<ColumnId, PredicateList>;
 
-    Status _parse_seek_range(const TabletReaderParams& read_params, std::vector<SeekRange>* ranges);
```

Comment:
why remove prefix _ of _parse_seek_range and _load_index and make these function public?

Reply:
Because when splitting the tablet by the key rowid ranges, we need parse the key range by using `parse_seek_range` , and load segment index to get the rowid range of the key range by using `load_index`.
However, when splitting the tablet, we haven't created the `TabletReader`. 
Therefore, we need make them public to use them outside `TabletReader`.